mongo/integration/csot_cse_prose_test.go

```diff
@@ -28,7 +28,7 @@ func TestCSOTClientSideEncryptionProse(t *testing.T) {
 	mt := mtest.New(t, mtest.NewOptions().MinServerVersion("4.2").CreateClient(false))
 	defer mt.Close()
 
-	mt.RunOpts("1. maxTimeMS is not set for commands sent to mongocryptd",
```

Comment:
What do these indices, such as "1. 2. 8." mean?

Reply:
As discussed offline, those indices refer to the numbering of the CSOT [prose tests](https://github.com/mongodb/specifications/tree/master/source/client-side-operations-timeout/tests) in the spec (this test is actually the second one listed under "Prose Tests". They should really be numbered in the spec, too, but I'll make a PR for that separately.